test/Microsoft.AspNetCore.Blazor.E2ETest/Tests/ComponentRenderingTest.cs

```diff
@@ -100,6 +100,9 @@ public void CanTriggerKeyPressEvents()
             Assert.Collection(liElements(),
                 li => Assert.Equal("a", li.Text),
                 li => Assert.Equal("b", li.Text));
+
+            // Textbox contains typed text
+            Assert.Equal("ab", inputElement.GetAttribute("value"));
```

Comment:
Note that this assertion would fail if the `event.preventDefault` call was still present.